src/bulkExport/stopExportJob.test.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import * as AWSMock from 'aws-sdk-mock';
+import AWS from 'aws-sdk';
+import { BulkExportStateMachineGlobalParameters } from './types';
+import { stopExportJobHandler } from './stopExportJob';
+
+AWSMock.setSDKInstance(AWS);
+
+describe('getJobStatus', () => {
```

Comment:
Should we cover the other two error cases? (bad env settings)

Reply:
added